### PR TITLE
pages: prepare what's new page

### LINKED ISSUE #766: pages: prepare what's new page
from https://github.com/zenodo/zenodo/wiki/What's-new%3F

### PR DESCRIPTION
**Pull request by slint**
* Adds a new What's New page. (closes #766)

Signed-off-by: Alexander Ioannidis <a.ioannidis@cern.ch>

### CODE REVIEW

File: zenodo/modules/theme/views.py [hunk 1]
```diff
@@ -35,3 +35,10 @@
     static_folder='static',
 )
 """Theme blueprint used to define template and static folders."""
+
+
+# FIXME: REMOVE THIS AND ITS TEMPLATE!!!
```
Comment (lnielsen) [inline zenodo/modules/theme/views.py]: I guess this shouldn't be here?

Comment (slint) [inline zenodo/modules/theme/views.py]: It's temporary, and just helps for quick prototyping. Will remove when finished

**Comment by lnielsen** [comment]
Can you also update the front-page (the box above "Using GitHub?") I was thinking to make similar to GItHub's announcement of new features on your profile page.

screen shot 2016-09-14 at 10 48 58

**Comment by slint** [comment]
whatsnew_front
whatsnew

**Pull request merged**